src/python/pants/testutil/pants_run_integration_test.py

```diff
@@ -357,6 +357,10 @@ def run_pants_with_workdir_without_waiting(self, command, workdir, config=None,
       env.update(extra_env)
     env.update(PYTHONPATH=os.pathsep.join(sys.path))
 
+    # Pants command that was called from the test shouldn't have a parent.
+    if env.get('PANTS_PARENT_BUILD_ID'):
```

Comment:
```suggestion
    if 'PANTS_PARENT_BUILD_ID' in env:
```

Seems slightly more semantically meaningful - we tend to use `.get` where we actually want to do something with the value, but I don't have a strong preference :)